scripts/primarybeamimage-fits.py

```diff
@@ -0,0 +1,33 @@
+import argparse
+import os
+import logging
+import numpy as np
+import primary_beam_correction as pbc
```

Comment:
Regarding imports, PEP 8 says
> Imports should be grouped in the following order:
>
>    Standard library imports.
>    Related third party imports.
>    Local application/library specific imports.
>
> You should put a blank line between each group of imports.

So add a blank line either side of the numpy import.

Reply:
Resolved.

Reply:
I will change the name after finishing with all other changes because i did not know how will git keep track of the changes made if the file is changed to a new name.